fastly/block_fastly_service_v1_logging_kafka.go

```diff
@@ -0,0 +1,264 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/fastly/go-fastly/fastly"
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var kafkaloggingSchema = &schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			// Required fields
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The unique name of the Kafka logging endpoint.",
+			},
+
+			"topic": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The Kafka topic to send logs to.",
+			},
+
+			"brokers": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "A comma-separated list of IP addresses or hostnames of Kafka brokers. Required.",
```

Comment:
Removed appended "Required".
```suggestion
				Description: "A comma-separated list of IP addresses or hostnames of Kafka brokers.",
```